generatorreceiver/topos/hipster_shop.yaml

```diff
@@ -37,35 +37,121 @@ topology:
             cluster_name: k8s-cluster-1
             request:
               cpu: 0.5
-              memory: 2048
             limit:
               cpu: 0.75
-              memory: 3072
-            restart:
-              every: 5m
           resourceAttrs:
             cloud.provider: aws
             cloud.region: us-west-2
+      metrics:
```

Comment:
This block of metrics was removed in main - please make sure they don't get added back. Probably a good idea to rebase your branch onto main.